swift_browser_ui_frontend/vite.config.js

```diff
@@ -101,6 +99,20 @@ const multipagePlugin = () => ({
   },
 });
 
+const htmlPlugin = (oidc) => {
+  if (oidc) {
+    return {
+      name: "html-transform",
+      transformIndexHtml: {
+        enforce: "pre",
+        apply: 'build',
```

Comment:
I think we don't want this `apply`, so that it will do the same also on the development server.